pkg/client/kubernetes/pods.go

```diff
@@ -67,6 +69,40 @@ func (c *Clientset) GetPodLogs(namespace, name string, podLogOptions *corev1.Pod
 	return buffer, nil
 }
 
+//ExecPod executes a command within container in a pod the command, usage example for command "ping -c 127.0.0.1"
+func (c *Clientset) ExecPod(namespace, name, containerName, command string) (io.Reader, error) {
```

Comment:
Why  does this function exist twice? https://github.com/gardener/gardener/pull/624/files#diff-23b959e447b752f833eca068de565193R14